.github/workflows/ci-maven.yaml

```diff
@@ -3,14 +3,18 @@ on:
     branches:
       - main
   pull_request:
+
 name: ci-maven
+
+env:
+  SHOWCASE_VERSION: 0.25.0
```

Comment:
Do we expect to manually change this version? A nice to have would be to set up renovate bot to automatically change it.

Reply:
Yes - this version will need to be manually changed (right now), and is coupled-but-distinct from [the selected showcase proto version](https://github.com/googleapis/gapic-generator-java/pull/1114/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5R126) used to generate the client.

Agreed this server version is a good candidate for renovate-bot, as we'll always be pointing to a release for its associated binary artifacts.

The client version would be a good candidate if there were consistent releases, but there aren't today - and we're currently needing to point to specific commits. I don't think we want renovate bot to be updating the client with each change to head.

- [ ] Use renovate bot to update the showcase server version.

Reply:
Based on our in-person discussion, I'm resolving this as a "nice to have but not required" feature.